packages/rs-dpp/src/lib.rs

```diff
@@ -65,6 +65,7 @@ pub mod prelude {
     pub type TimestampMillis = u64;
     pub type Revision = u64;
     pub type IdentityNonce = u64;
+    pub type FeeMultiplier = u16;
```

Comment:
So I can only double or more processing fees? I can't add just a little to win other STs?

Reply:
No, the fee multiplier used here is just a percentage add.

Reply:
I'm renaming this to UserFeeMultiplier to avoid the confusion you had.

Reply:
I would like to see some documentation on this type or somewhere else in code (maybe I just haven't seen it) that says that a FeeMultiplier is a percentage increase, such that the value of 0 results in no increase; a value of 100 results in a 1% increate and a value of 10000 results in a doubling; assuming my understanding is correct?

Reply:
@thephez For the future this UserFeeIncrease is the additional percentage of the processing fee. So a 1 here means we pay 1% more in processing fees. a 100 means we pay 100% more.

Reply:
Why not add a comment to code here?

Reply:
Done